bricks-maven-plugin/src/test/java/com/github/ingogriebsch/bricks/maven/plugin/GenerateComponentMojoTest.java

```diff
@@ -34,21 +34,21 @@
     @Rule
     public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-    @Test
-    public void execute_should_create_file() throws Exception {
-        File outputDirectory = temporaryFolder.getRoot();
-        String outputFilename = "bricks.json";
-
-        Component component = new Component();
-        component.setId("id");
-
-        GenerateComponentMojo mojo = new GenerateComponentMojo();
-        mojo.setOutputDirectory(outputDirectory);
-        mojo.setOutputFilename(outputFilename);
-        mojo.setComponent(component);
-        mojo.execute();
-
-        assertThat(new File(outputDirectory, outputFilename)).exists().isFile();
-    }
+    // @Test
+    // public void execute_should_create_file() throws Exception {
+    // File outputDirectory = temporaryFolder.getRoot();
+    // String outputFilename = "bricks.json";
+    //
+    // Component component = new Component();
+    // component.setId("id");
+    //
+    // GenerateComponentMojo mojo = new GenerateComponentMojo();
+    // mojo.setOutputDirectory(outputDirectory);
+    // mojo.setOutputFilename(outputFilename);
+    // mojo.setComponent(component);
+    // mojo.execute();
+    //
+    // assertThat(new File(outputDirectory, outputFilename)).exists().isFile();
+    // }
 
 }
```

Comment:
The whole test file can be removed if we remove the source file (which we do).

Reply:
sure, missed that